app/assets/stylesheets/style.scss

```diff
@@ -10287,15 +10288,10 @@ background: linear-gradient(180deg, rgba(255,252,252,0) 0%, rgba(126,109,109,0.5
   z-index: 2;
 }
 .notice{
-  background-color:black;
-  color:#e9c318;
-  border: 1px solid;
-  border-color:black;
-  font-family: "Arial";
+  color:#EE9D1C;
   font-weight: bold;
-  font-style: italic;
-  padding-left: 40px;
 }
+
```

Comment:
Notice now appears in the nav and doesn't require all this fancy ish

Reply:
clean. nice